lib/galaxy/util/script.py

```diff
@@ -26,30 +26,43 @@ def main(argv=None):
     if argv is None:
         argv = sys.argv[1:]
     args = _arg_parser().parse_args(argv)
-    kwargs = _app_properties(args)
+    kwargs = app_properties_from_args(args)
     action = args.action
     action_func = ACTIONS[action]
     action_func(args, kwargs)
 
 
-def _app_properties(args):
-    # FIXME: you can use galaxy.util.path.extensions for this
-    config_file = args.config_file or find_config_file(args.app)
+def app_properties_from_args(args, legacy_config_override=None):
+    config_file = config_file_from_args(args, legacy_config_override=legacy_config_override)
     app_properties = load_app_properties(config_file=config_file, config_section=args.config_section)
     return app_properties
 
 
+def config_file_from_args(args, legacy_config_override=None):
+    # FIXME: you can use galaxy.util.path.extensions for this
```

Comment:
I think this comment is outdated, `galaxy.util.path.extensions` is used in `find_config_file()`.